docs/content/guides/dagster/using-environment-variables-and-secrets.mdx

```diff
@@ -13,11 +13,23 @@ description: Learn how to securely define and access environment variables in yo
 
 How environment variables are declared depends on how your Dagster project is deployed.
 
-### Dagster Cloud
+<TabGroup>
+  <TabItem name="Local development">
 
-Environment variables can be set a variety of ways in Dagster Cloud. Refer to the [Dagster Cloud environment variables guide](/dagster-cloud/developing-testing/environment-variables-and-secrets) for more info.
+If your Dagster project contains a `.env` file, Dagster will automatically load the environment variables it contains when Dagit is started.
```

Comment:
the .env file thing only works starting in 1.1 (not sure if we need to call that out or the docs versioning is enough)

Reply:
It doesn't hurt to be explicit about it 👍